web_src/js/htmx.js

```diff
@@ -0,0 +1,20 @@
+import * as htmx from 'htmx.org';
+import {showErrorToast} from './modules/toast.js';
+
+window.htmx = htmx;
```

Comment:
Hmm ... wait, why it needs to be exposed to global `window` ?

Reply:
I actually think its not needed. For plugins `htmx` needs to be globally available, however it seems this needs to be done using something like: https://github.com/go-gitea/gitea/blob/7cf50ef276edceda44e332ced694e3070c7a60db/webpack.config.js#L182

Reply:
I followed https://htmx.org/docs/#webpack

Reply:
@yardenshoham ping ;)

Reply:
Ok everything works without it so I removed it https://github.com/go-gitea/gitea/pull/28908/commits/ae3b58f193bc04b65ad8e69a2b2acdc405f5d5e3